src/SegmentedMsg.chpl

```diff
@@ -917,20 +917,20 @@ module SegmentedMsg {
   
   proc registerMe() {
     use CommandMap;
-    registerFunction("segmentLengths", segmentLengthsMsg);
-    registerFunction("segmentedHash", segmentedHashMsg);
-    registerFunction("segmentedSearch", segmentedSearchMsg);
-    registerFunction("segmentedFindLoc", segmentedFindLocMsg);
-    registerFunction("segmentedFindAll", segmentedFindAllMsg);
-    registerFunction("segmentedPeel", segmentedPeelMsg);
-    registerFunction("segmentedSub", segmentedSubMsg);
-    registerFunction("segmentedIndex", segmentedIndexMsg);
-    registerFunction("segmentedBinopvv", segBinopvvMsg);
-    registerFunction("segmentedBinopvs", segBinopvsMsg);
-    registerFunction("segmentedGroup", segGroupMsg);
-    registerFunction("segmentedIn1d", segIn1dMsg);
-    registerFunction("randomStrings", randomStringsMsg);
-    registerFunction("segStr-assemble", assembleStringsMsg);
+    registerFunction("segmentLengths", segmentLengthsMsg, getModuleName());
+    registerFunction("segmentedHash", segmentedHashMsg, getModuleName());
+    registerFunction("segmentedSearch", segmentedSearchMsg, getModuleName());
+    registerFunction("segmentedFindLoc", segmentedFindLocMsg, getModuleName());
+    registerFunction("segmentedFindAll", segmentedFindAllMsg, getModuleName());
+    registerFunction("segmentedPeel", segmentedPeelMsg, getModuleName());
+    registerFunction("segmentedSub", segmentedSubMsg, getModuleName());
+    registerFunction("segmentedIndex", segmentedIndexMsg, getModuleName());
+    registerFunction("segmentedBinopvv", segBinopvvMsg, getModuleName());
+    registerFunction("segmentedBinopvs", segBinopvsMsg, getModuleName());
+    registerFunction("segmentedGroup", segGroupMsg, getModuleName());
+    registerFunction("segmentedIn1d", segIn1dMsg, getModuleName());
+    registerFunction("randomStrings", randomStringsMsg, getModuleName());
+    registerFunction("segStr-assemble", assembleStringsMsg, getModuleName());
```

Comment:
I don't know how much Reflection costs, so if it's negligible dis-regard:  You could save `getModuleName()` at the beginning of `registerMe` and pass the string instead of calling the function every time.  i.e.
```chapel
const mod_name = getModuleName();
registerFunction("cmd1", myFunc1, mod_name);
registerFunction("cmd2", myFunc2, mod_name);
...
```


Also, separately, why did `registerBinaryFunction` below not get the same treatment?

Reply:
Ya, I will add it to `registerBinaryFunction` as well

Reply:
The reason that I left the overload of `registerFunction` was that I didn't want `arkouda_server` and other base modules to be added to the list of output modules, since that would not be a valid configuration file. I suppose we could instead just check if it is `arkouda_server` or in a set of the base modules or something, but I think I prefer the overload as it is today.